recipes/miniwdl/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "miniwdl" %}
+{% set version = "0.1.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "fbb646a09af08f0d13f2f197eba01db576a70745190fa8546d50d165f21682d4"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - pbr
+    - python >=3.6
+    - setuptools
+    - docker-py
+    - lark-parser 0.7.1
+
+  run:
+    - python >=3.6
+    - setuptools
+    - docker-py
+    - lark-parser 0.7.1
+test:
+  imports:
+    - WDL
+    - WDL.Env
+    - WDL.Error
+    - WDL.Expr
+    - WDL.StdLib
+    - WDL.Tree
+    - WDL.Type
+    - WDL.Value
+    - WDL.Walker
+
+about:
+  home: https://github.com/chanzuckerberg/miniwdl"
```

These don't need to be quoted but it doesn't matter either way, just fixing this one as it was misbalenced:
```suggestion
  home: https://github.com/chanzuckerberg/miniwdl
```